torchci/rockset/commons/__sql/weekly_force_merge_stats.sql

```diff
@@ -0,0 +1,174 @@
+-- gets percentage of total force merges, force merges with failures, and force merges without failures (impatient)
```

Comment:
Kudos to the comment, may be also add a few lines about where this is used on HUD